coops-ui/src/views/AnnualReport.vue

```diff
@@ -503,6 +506,16 @@ export default {
       this.toggleFiling(modified ? 'add' : 'remove', 'OTFDR')
     },
 
+    getSectionCode () :string {
+      switch (this.entityType) {
+        case EntityTypes.Coop:
```

Comment:
Check with Cam on his change to how constants are imported and used. If his PR goes in first then you will have to update this code. If your PR goes in first then he will have to update this code.

Reply:
I think this comment is obsolete. I believe EntityTypes is still an enum (not a bunch of constants) so this code is still OK. Check with Cam anyway :)